README.md

```diff
@@ -39,16 +47,106 @@ curl http://admin:admin@nextcloud.local/ocs/v1.php/cloud/capabilities\?format\=j
 }
 ```
 
-### Fetching lock details
+### WebDAV: Fetching lock details
 
 WebDAV returns the following additional properties if requests through a `PROPFIND`:
 
 - `{http://nextcloud.org/ns}lock`: `true` if the file is locked, otherwise `false`
+- `{http://nextcloud.org/ns}lock-owner-type`: User id of the lock owner
+  - `0` represents a manual lock by a user
+  - `1` represents a collaboratively locked file, e.g. when being edited through Office or Text
+  - `2` represents a WebDAV lock identified by a lock token, which will have the other properties set as if it was type 0
 - `{http://nextcloud.org/ns}lock-owner`: User id of the lock owner
 - `{http://nextcloud.org/ns}lock-owner-displayname`: Display name of the lock owner
+- `{http://nextcloud.org/ns}lock-owner-editor`: App id of an app owned lock to allow clients to suggest joining the collaborative editing session through the web or direct editing
 - `{http://nextcloud.org/ns}lock-time`: Timestamp of the log creation time
+- `{http://nextcloud.org/ns}lock-timeout`: TTL of the lock in seconds staring from the creation time
+- `{http://nextcloud.org/ns}lock-token`: Unique lock token (to be preserved on the client side while holding the lock to sent once full webdav locking is implemented)
+
+```bash
+curl -X PROPFIND \
+  --url http://admin:admin@nextcloud.dev.local/remote.php/dav/files/admin/myfile.odt \
+  --data '<D:propfind xmlns:D="DAV:" xmlns:NC="http://nextcloud.org/ns">
+	<D:prop>
+		<NC:lock />
+		<NC:lock-owner-type />
+		<NC:lock-owner />
+		<NC:lock-owner-displayname />
+		<NC:lock-owner-editor />
+		<NC:lock-time />
+	</D:prop>
+</D:propfind>'
+```
+
+
+### WebDAV: Manually lock a file
+
+```
+curl -X LOCK \
+  --url http://admin:admin@nextcloud.dev.local/remote.php/dav/files/admin/myfile.odt \
+  --header 'X-User-Lock: 1'
```

Comment:
Maybe we should explain that this specifies the lock type. It's pretty self explanatory but still.